From 93cf25ba663e68a6a6f4237fbe0ef8349b3f37ef Mon Sep 17 00:00:00 2001
From: Sumit Garg <sumit.garg@linaro.org>
Date: Fri, 14 Feb 2020 07:40:11 +0000
Subject: [PATCH] Allow to build with OE LLVM cross compiled package

The default LLVM cmake package requires all libraries, headers and tools
to be present but in case of cross compilation, OE only provides
target specific libraries and headers and requires native llvm tools.

So instead of looking for a complete llvm package, look for libraries and
headers which are installed in target sysroot-recipe path.

Signed-off-by: Sumit Garg <sumit.garg@linaro.org>
---
 CMakeLists.txt | 2 --
 1 file changed, 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 7bd0f3b2..6f1f12c0 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -39,8 +39,6 @@ endif()
 if(NOT PYTHON_ONLY AND ENABLE_CLANG_JIT)
 find_package(BISON)
 find_package(FLEX)
-find_package(LLVM REQUIRED CONFIG)
-message(STATUS "Found LLVM: ${LLVM_INCLUDE_DIRS} ${LLVM_PACKAGE_VERSION}")
 find_package(LibElf REQUIRED)

 # clang is linked as a library, but the library path searching is
--
2.17.1
